app/src/main/java/com/example/calculator/MainActivity.kt

```diff
@@ -1,11 +1,144 @@
 package com.example.calculator
 
-import androidx.appcompat.app.AppCompatActivity
 import android.os.Bundle
+import android.text.TextUtils
+import android.util.Log
+import android.view.View
+import android.widget.Button
+import androidx.appcompat.app.AppCompatActivity
+import com.example.calculator.databinding.ActivityMainBinding
+
+class MainActivity : AppCompatActivity() , View.OnClickListener{
+
+    private lateinit var binding: ActivityMainBinding
+    private var operandStr = StringBuilder("")
+    private var operand: String = ""
+    private var operandListDefault = arrayListOf("+" , "-" , "x" , "/")
+    private var numberList = arrayListOf("0" , "1" , "2" , "3" , "4" , "5" , "6" , "7" , "8" , "9")
```

Comment:
https://kotlinlang.org/docs/coding-conventions.html#immutability
list này có cần thay đổi nữa không nêus không thì để val 
```suggestion
    private val numbers = listOf("0" , "1" , "2" , "3" , "4" , "5" , "6" , "7" , "8" , "9")
```